src/coffee/services/product-projections.coffee

```diff
@@ -65,6 +65,22 @@ class ProductProjectionService extends BaseService
     debug 'setting fuzzy: %s', fuzzy
     this
 
+  # Public: Define whether to mark product variants in the search result matching the criteria or not.
+  #
+  # markMatchingVariants - {Boolean} `true` to do a mark product variants, `false` not mark matching product variants
+  #
+  # Returns a chained instance of `this` class
+  #
+  # Examples
+  #
+  #   service = client.productProjections
+  #   service.text('Sapphure', 'en').markMatchingVariants(true).search()
+  markMatchingVariants: (value) ->
+    return this unless _.isBoolean(value)
```

Comment:
Returns `this` if the value passed in is not a `value`. This enforces the usage of just `true` or `false`